app-thunderbird/src/main/res/values/constants.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
     <string name="app_webpage_url" translatable="false">https://k9mail.app/</string>
     <string name="user_forum_url" translatable="false">https://support.mozilla.org/products/thunderbird-android</string>
-    <string name="donate_url" translatable="false">https://www.thunderbird.net/donate/</string>
+    <string name="funding_url" translatable="false">https://www.thunderbird.net/donate/mobile/?form=tfa</string>
```

Comment:
Why not remove this one? It overwrites the string from `:legacy:ui` with the same value.